schema.graphql

```diff
@@ -16,3 +16,97 @@ type GaugeStakingHistory @entity {
   lastBlockNumber_: BigInt!
   lastBlockTimestamp_: BigInt!
 }
+
+type Builder @entity(immutable: false) {
+  id: Bytes!
+  gauge_: Bytes!
+  backerRewardPercentage_: BigInt!
+  rewardReceiver_: Bytes!
+  state_: BuilderState @derivedFrom(field: "builder_")
+  isHalted_: Boolean!
```

Comment:
To include the handlers needed to set this field. We could do it in a different task

Reply:
updated